setup.py

```diff
@@ -15,14 +15,10 @@
     ],
     extras_require={
         'dev': [
-            "coverage==4.0a1",
-            "flake8==2.1.0",
-            "mccabe==0.3",
-            "pep8==1.6.1",
-            "pyflakes==0.8.1",
-            "pylint==1.0.0",
-            "pytest==2.4.2",
-            "pytest-timeout==0.4",
+            "coverage==4.0.3",
```

Comment:
We could add the latest versions of `flake8` (`2.5.4`), `pylint` (`1.5.5`) and `pytest` (`2.9.1`) if there isn't a reason to use the current versions.